### DISCUSSION TITLE
Add ability to Export/Save/Share receive address QR Code

### DISCUSSION THREAD

**Discussion by UmbahaSpodPaha** [iotaledger/firefly#1563] [Features / Enhancements]
## Description
Add ability to Export/Save/Share receive address QR Code

## Motivation
I think it's pretty obvious function to have with QR Code. We can share it with friends etc.

## Requirements
Being able to share/save/export QR code

## Are you planning to do it yourself in a pull request?
No

**Comment by rafaeldjpbrochado** [comment]
Great idea.

**Comment by maxwellmattryan** [comment]
Hey @UmbahaSpodPaha, I've moved this into a discussion [here](https://github.com/iotaledger/firefly/discussions/1484) to close this issue 👍
